pgmpy/tests/help_functions.py

```diff
@@ -5,3 +5,8 @@ def recursive_sorted(li):
     for i in range(len(li)):
         li[i] = sorted(li[i])
     return sorted(li)
+
+def convert_to_frozenset(edges) :
```

Comment:
@VSSILPA I think we can have a better name for this function. Also IMO it should return a frozenset of frozenset.
So function name could be recursive_frozenset, if you can think of some name, please use that.